packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -178,6 +244,11 @@ export default declare<State>((api) => {
               );
             }
           }
+
+          // Build our `state.compiledImports[…]` state now that we've traversed JSX pragmas
+          path.traverse<State>(buildCompiledImportsCache, this);
+          // Build our `state.cssMap[…]` before we traverse the JSXElements which consume it
+          path.traverse<State>(buildCssMapState, this);
```

Comment:
I don't know if this is the best way to guarantee the order of things—just stuffing them into `Program.enter`, but it appears to work.

This could definitely be improved and traversion could be reduced by 1-5x if desired…

Reply:
This is wrong…in tests and other places, these aren't guaranteed to be at the Program scope, so this breaks our own tests.